installer/install-all-components.sh

```diff
@@ -0,0 +1,123 @@
+kubectl create ns cloudflow
```

Comment:
This should be deleted. We don't want to go back to a bash script for install. The installer directory will be removed.

Reply:
This is a utility for us to spin up our environment to run the integration tests. I find it quite useful. I'll remove it from git then.